index.html

```diff
@@ -11,6 +11,7 @@
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
     <script src="https://kit.fontawesome.com/e1cc58c746.js" crossorigin="anonymous"></script>
     <script src="https://kit.fontawesome.com/e1cc58c746.js" crossorigin="anonymous"></script>
+    <script src="https://kit.fontawesome.com/e1cc58c746.js" crossorigin="anonymous"></script>
 </head>
 <body>
     
```

Comment:
Please add linter for HTML, CSS, JavaScript from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) linter config.